test/urlForHotspotImage.test.ts

```diff
@@ -1,7 +1,14 @@
 import urlForImage from '../src/urlForImage'
 import {croppedImage, materializedAssetWithCrop, noHotspotImage, uncroppedImage} from './fixtures'
 
-describe('urlForHotspotImage', () => {
+describe('urlForImage', () => {
+  test('should throw on invalid source', () => {
+    // @ts-ignore: Because we're throwing on invalids
```

Comment:
Looking forward to be able to use `//@ts-expect-error` in cases like these 💯 https://devblogs.microsoft.com/typescript/announcing-typescript-3-9-beta/#ts-expect-error-comments